### PR TITLE
Help text/README updates, Additional minor fixes

### PR DESCRIPTION
**Pull request by pranavchary**
- Updated text output by `/mr-helper help` to be useful to users
- Updated README.md
- Restrictions added to certain input fields
  - Minimum value for simulated keystone level and goal M+ rating
  - Minimum and maximum length for character names
- Sanitize realm name if provided to account for apostrophes and spaces (eg. Kel'Thuzad, Area 52)
- Added helper function to capitalize words

### CODE REVIEW

File: README.md
Comment (KnobleChipmunk) [inline README.md]: I'd suggest adding the server name. ie- 

> for the World of Warcraft guild The Last Dynasty (US - Thrall), **TLD Reforged**.

**Review by KnobleChipmunk — DISMISSED** [review]

File: README.md
Comment (pranavchary) [reply README.md]: Added the Discord server name as suggested. It'll need re-approval to merge since the change was made after your approval. Sorry 😅

**Review by KnobleChipmunk — APPROVED** [review]

**Pull request merged**
